src/main/scala/uk/gov/nationalarchives/tdr/api/auth/AuthorisationTags.scala

```diff
@@ -182,3 +183,16 @@ object ValidateHasExportAccess extends SyncAuthorisationTag {
     }
   }
 }
+
+object ValidateHasReportingAccess extends SyncAuthorisationTag {
```

Comment:
Will we have to add this permission to the `getConsignment` endpoint as well?

Reply:
If Phil will be using the `getConsignment` endpoint then I guess so.

Reply:
Phil still wants to use the `getConsignment` endpoint

Reply:
Spoke to Phil. Will go with just access to the `consignments` endpoint for now. Will add access to `getConsignment` if it appears he will need it later.